feature/experimental/system/health/tests/system_generic_health_check/system_generic_health_check_test.go

```diff
@@ -0,0 +1,737 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package system_generic_health_check_test
+
+import (
+	"context"
+
+	"regexp"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/components"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/raw"
+	"github.com/openconfig/ygnmi/ygnmi"
+
+	fpb "github.com/openconfig/gnoi/file"
+	hpb "github.com/openconfig/gnoi/healthz"
+	tpb "github.com/openconfig/gnoi/types"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+var (
+	vendorCoreFilePath = map[ondatra.Vendor]string{
+		ondatra.JUNIPER: "/var/core/",
+		ondatra.CISCO:   "/misc/disk1/",
+		ondatra.NOKIA:   "/var/core/",
+	}
+	vendorCoreFileNamePattern = map[ondatra.Vendor]*regexp.Regexp{
+		ondatra.JUNIPER: regexp.MustCompile(".*.tar.gz"),
+		ondatra.CISCO:   regexp.MustCompile("/misc/disk1/.*core.*"),
+		ondatra.NOKIA:   regexp.MustCompile("/var/core/coredump-.*"),
+	}
+)
+
+const (
+	cpuType               = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CPU
+	lineCardType          = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_LINECARD
+	fabricCardType        = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_FABRIC
+	controllerCardType    = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD
+	cpuHighUtilization    = 80
+	memoryHighUtilization = 80
+	threshold             = uint8(cpuHighUtilization)
```

Comment:
Thanks for your comments. Because the test doesn't compare values now, removed the variables *threshold*, *cpuHighUtilization*, *memoryHighUtilization*.